hack/jjb_template.jinja2

```diff
@@ -50,6 +50,7 @@
         {% elif 'daily-ocp4.9-powervm-p10-min' in JOB_NAME %}"00 17 * * *  "
         {% elif 'daily-ocp4.10-powervm-p10-min' in JOB_NAME %}"00 21 * * *  "
         {% elif 'daily-ocp4.10-powervm-p10-ssp-min' in JOB_NAME %}"00 08 * * 2,5 "
+        {% elif 'daily-ocp4.10-powervm-p10-ovn-min' in JOB_NAME %}"00 08 * * 1,3"
```

Comment:
lets schedule this daily `{% elif 'daily-ocp4.10-powervm-p10-ovn-min' in JOB_NAME %}"00 01 * * * "`